pkg/schemas/stream_gen.go

```diff
@@ -86,13 +91,22 @@ func StreamToSchema(stream *sdk.Stream) map[string]any {
 		streamSchema["comment"] = stream.Comment
 	}
 	if stream.TableName != nil {
-		streamSchema["table_name"] = stream.TableName
+		tableId, err := sdk.ParseSchemaObjectIdentifier(*stream.TableName)
```

Comment:
question: were this and the next added manually or is our generator already that smart? (if manually, please leave a comment so that we don't overwrite it

Reply:
It was changed manually, I added a comment. Note that we don't mark the manual changes after generation.

Reply:
Close after reading.